README.md

```diff
@@ -175,7 +175,11 @@ Depending on the distribution you use, it might be possible to install lldap
 from a package repository, officially supported by the distribution or
 community contributed.
 
-#### Arch Linux
+##### Debian, CentOS Fedora, OpenSUSE, Ubuntu
+The pre-packaged per distribution can be fetched [LLDAP OBS](https://software.opensuse.org//download.html?project=home%3AMasgalor%3ALLDAP&package=lldap)
```

Comment:
Small nits on the formulation:

The package for these distributions can be found at LLDAP OBS (link).
When using the distributed package, the default login is `admin`/`password`. You can change that from the web UI after starting the service.